### PR TITLE
Fix backup restore timer metrics

### PR DESCRIPTION
**Pull request by LuQQiu**
Previous we record the time of last backup/restore using Timer metrics.
Timer metrics has some problem of recording operation durations. 
In local backup restore tests, timer metrics cannot record the restore time correctly.
This PR change to use Gauge metrics.

### CODE REVIEW

**Comment by LuQQiu** [comment]
@calvinjia  PTAL, thanks!

File: core/server/common/src/main/java/alluxio/master/BackupManager.java [hunk 1]
```diff
@@ -76,6 +75,9 @@
   private long mBackupEntriesCount;
   private long mRestoreEntriesCount;
 
+  private long mBackupTimeMs;
+  private long mRestoreTimeMs;
```
Comment (calvinjia) [inline core/server/common/src/main/java/alluxio/master/BackupManager.java]: Should we set them to `-1` to indicate there was no backup or restore completed?

Comment (LuQQiu) [reply core/server/common/src/main/java/alluxio/master/BackupManager.java]: Good idea! Set it to -1

**Comment by calvinjia** [comment]
alluxio-bot, merge this please?

**Pull request merged**